psamm/fastgapfill.py

```diff
@@ -0,0 +1,94 @@
+# This file is part of PSAMM.
+#
+# PSAMM is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PSAMM is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PSAMM.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Copyright 2016  Jon Lund Steffensen <jon_steffensen@uri.edu>
+# Copyright 2016  Chao Liu <lcddzyx@gmail.com>
+
+from __future__ import unicode_literals
+
+import logging
+
+from six import iteritems
+from .fastcore import fastcore
+
+logger = logging.getLogger(__name__)
+
+
+def create_model_complete(model, db_weight=None, ex_weight=None,
+                          tp_weight=None, penalties=None):
+    """create a complete metabolic model for fastcore algorithm.
```

Comment:
Capitalize first letter of sentence. Also, I think this function has to be explained in terms of the fastgapfill procedure described in the paper, i.e. describe that the function creates a MetabolicModel with all reactions added (the reaction database in the model is taken to be the universal database) and also with artificial exchange and transport reactions added.